### ISSUE TITLE
IAF 7.7-20211223.143851 certificate error

### ISSUE THREAD

**Issue by jeri-nngroup** [ibissource/iaf#2596]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
With IAF 7.7-20211223.143851 I'm getting an error on a request from ibis4oma > engageone, where I'm getting a complaint about an inability to create a valid cert path. This does not happen on 7.7-20211214.123740 where the IAF update was the only change I've done.

Resulting stacktrace starts with:
nl.nn.adapterframework.core.SenderException: (SSLHandshakeException): (j) PKIX path building failed: (IBMCertPathBuilderException) unable to find valid certification path to requested target
at nl.nn.adapterframework.http.HttpSenderBase.sendMessage(HttpSenderBase.java:675)

Full stacktrace is attached.

**Reporter**
Tjalsma, J. (Jeri) <jeri.tjalsma@nn-group.com>

1. Update to the mentioned IAF version,
2. Send a message to the DMOD adapter (ladybug report can be shared privately as it can contain passwords)
3. See error.

**Screenshots**

**Configuration**
```
Realtime config, Pipeline DocumentOnDemand, Pipe SendtoEngageOne_WSS
```

**Input**
```
Please contact met for the ladybug report
```

**Environment**
```
go to https://***/ibis4oma/iaf/gui/#!/environment-variables, this happens on ibis4oma SI on T, but can't share server details.
```

**Additional Environment**
 n/a

**Additional Context**
[nl.nn.adapterframework.core.SenderExcept.log](https://github.com/ibissource/iaf/files/7813676/nl.nn.adapterframework.core.SenderExcept.log)

**Comment by gvanbrakel** [comment]
This could likely be a duplicate of issue #2582, that has been fixed #2583. Could you please retest with the latest version?

**Comment by jeri-nngroup** [comment]
Thanks for the feedback! I've upgraded to 7.7-20220104.143424 and with that version the error no longer occurs.
